<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainBox</class>
 <widget class="QWidget" name="MainBox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>471</width>
    <height>207</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QWidget" name="widget_serial" native="true"/>
   </item>
   <item>
    <widget class="QWidget" name="widget_visual" native="true"/>
   </item>
   <item>
    <widget class="QWidget" name="widget_grapher" native="true"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
